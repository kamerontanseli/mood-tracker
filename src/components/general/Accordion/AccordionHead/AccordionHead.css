.AccordionHead {
  display: flex;
  align-items: center;
}