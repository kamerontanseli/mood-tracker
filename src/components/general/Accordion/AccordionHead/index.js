import "./AccordionHead.css";
import React from "react";

const AccordionHead = ({ children }) => (
  <div className="AccordionHead">{children}</div>
);

export default AccordionHead;
