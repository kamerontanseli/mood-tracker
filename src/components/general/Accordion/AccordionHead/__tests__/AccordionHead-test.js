import React from 'react'
import renderer from 'react-test-renderer'
import AccordionHead from '..'

describe('render', () => {
  it('should render', () => {
    const snapshot = renderer.create(
      <AccordionHead>Hello world</AccordionHead>
    );
    expect(snapshot).toMatchSnapshot()
  })
})