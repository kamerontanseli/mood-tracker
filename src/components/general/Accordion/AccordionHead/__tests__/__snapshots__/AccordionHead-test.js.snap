// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render should render 1`] = `
<div
  className="AccordionHead"
>
  Hello world
</div>
`;
